hardware/Brakelight.PrjPcb

```diff
@@ -35,6 +35,12 @@ OutputPath=
 LogFolderPath=
 ManagedProjectGUID=
 IncludeDesignInRelease=0
+CrossRefSheetStyle=1

```

Comment:
This shouldn’t be in this PR. You may need to rebase. Same with the other files in /hardware/